internal/commands/report_test.go

```diff
@@ -36,12 +36,13 @@ func testReportCommand(t *testing.T, when spec.G, it spec.S) {
 	it.Before(func() {
 		var err error
 		logger = ilogging.NewLogWithWriters(&outBuf, &outBuf)
-		command = commands.Report(logger, testVersion)
 
 		tempPackHome, err = ioutil.TempDir("", "pack-home")
 		h.AssertNil(t, err)
 
 		packConfigPath = filepath.Join(tempPackHome, "config.toml")
+		cmd = commands.Report(logger, testVersion, packConfigPath)
+		cmd.SetArgs([]string{})
```

Comment:
The failure to set args was causing issues with my local tests, so I added it to ensure that it passes, and took the opportunity to do dependency injection. I can revert that if requested, though.